commands/management/add.go

```diff
@@ -38,6 +39,10 @@ func addGlobalFlags(cmd *kingpin.CmdClause) {
 	cmd.Flag("port", "Service port number (overrides positional argument)").Uint16Var(&addPortFlag)
 }
 
+func addLogLevelFlag(cmd *kingpin.CmdClause) {
+	cmd.Flag("log-level", "Service logging level").Default("warn").EnumVar(&addLogLevel, "debug", "info", "warn", "error", "fatal")
+}
+
```

Comment:
Why not to make it part of `addGlobalFlags`?

Reply:
> Why not to make it part of `addGlobalFlags`?

Because `addLogLevelFlag` possible will used in inventory too
And because all global flags used in processGlobalAddFlagsWithSocket and i don't want to mix

Reply:
I see that `addGlobalFlags` and `addLogLevelFlag` always used together. If we will need this flag somewhere else we always can refactor it later (you will need to refactor it anyway because the function is private).

Reply:
@artemgavrilov okey